doc/user_guide/eds.rst

```diff
@@ -512,11 +512,19 @@ The background can be subtracted from the X-ray intensities with the :py:meth:`~
 Quantification
 --------------
 
-.. versionadded:: 0.8
+Hyperspy now includes three methods for EDS quantification; Cliff-Lorimer, zeta-factors and ionization cross sections.
+
+Quantification can be applied to the background subtracted intensities with the :py:meth:`~._signals.eds_tem.EDSTEMSpectrum.quantification` method. These instensities are a stack of images for each element which can be extracted using :py:meth:`~._signals.eds.EDSSpectrum.get_lines_intensity`. The quantification method, needs be specified as either 'CL', 'zeta', or 'cross_section'. If no method is specified the function will raise an exception.
```

Comment:
Perhaps word to emphasise that if you want to use background subtracted intensities, you must first do a background subtraction... We don't want people thinking that's done by default.